hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -104,8 +116,12 @@
     MESSAGES_LOADING_WAIT_FOR_TOPICS_CACHE("frontend.messages.loading.wait.for.topics.cache", 10),
     MESSAGES_LOADING_WAIT_FOR_BROKER_TOPIC_INFO("frontend.messages.loading.wait.for.broker.topic.info", 5),
 
-    CONSUMER_COMMIT_OFFSET_PERIOD("consumer.commit.offset.period", 15),
-    CONSUMER_COMMIT_OFFSET_QUEUES_SIZE("consumer.commit.offset.queues.size", 500_000),
+    CONSUMER_RECEIVER_POOL_TIMEOUT("consumer.receiver.pool.timeout", 100),
+    CONSUMER_RECEIVER_READ_QUEUE_CAPACITY("consumer.receiver.read.queue.capacity", 1000),
+
+    CONSUMER_COMMIT_OFFSET_PERIOD("consumer.commit.offset.period.ms", 15 * 1000),
+    CONSUMER_COMMIT_OFFSET_QUEUES_SIZE("consumer.commit.offset.queues.size", 1000),
```

Comment:
What if in the meantime consumer gets restarted? The message is already committed, so it will not be retried anymore and we will break our contract with the user. It might be, that it will be retried for 5 minutes instead of an hour.